tests/data/instance_test.py

```diff
@@ -40,3 +41,23 @@ def test_duplicate(self):
         instance.add_field("labels", LabelField("some_label"))
         assert "labels" not in other.fields
         assert other != instance  # sanity check on the '__eq__' method.
+
+    def test_human_readable_repr(self):
+        words_field = TextField([Token("hello")], {})
+        label_field = LabelField(1, skip_indexing=True)
+        instance1 = Instance({"words": words_field, "labels": label_field})
+
+        assert type(instance1.human_readable_dict()) is dict
+        assert instance1.human_readable_dict() == {"words": ["hello"], "labels": 1}
+
+        instance1_human_readable_dict = instance1.human_readable_dict()
+        array = TensorField(numpy.asarray([1.0, 1, 1]))
+        array_human_readable_dict = {
+            "shape": [3],
+            "element_mean": 1.0,
+            "element_std": 0,
+            "type": "torch.float64",
```

Comment:
Oh, I didn't realize we'd get the "torch." in the dtype. I think it would be better if we just saw:

```suggestion
            "type": "float64",
```

So maybe in `TensorField.human_readable_repr()` we do `str(tensor.dtype).replace("torch.", "")` or something